src/main/kotlin/com/vk/kphpstorm/inspections/KphpStrictTypesEnableInspection.kt

```diff
@@ -0,0 +1,44 @@
+package com.vk.kphpstorm.inspections
+
+import com.intellij.codeInspection.ProblemHighlightType
+import com.intellij.codeInspection.ProblemsHolder
+import com.intellij.psi.PsiElementVisitor
+import com.jetbrains.php.lang.documentation.phpdoc.psi.impl.PhpDocCommentImpl
+import com.jetbrains.php.lang.inspections.PhpInspection
+import com.jetbrains.php.lang.psi.elements.Declare
+import com.jetbrains.php.lang.psi.elements.PhpPsiElement
+import com.jetbrains.php.lang.psi.elements.impl.DeclareImpl
+import com.jetbrains.php.lang.psi.visitors.PhpElementVisitor
+import com.vk.kphpstorm.inspections.quickfixes.AddStrictTypesCommentQuickFix
+
+class KphpStrictTypesEnableInspection : PhpInspection() {
+    val strictTypeEnable: CharSequence = "strict_types=1"
+    val kphpTagStrictTypeEnable: CharSequence = "@kphp-strict-types-enabled"
+
+    override fun buildVisitor(holder: ProblemsHolder, isOnTheFly: Boolean): PsiElementVisitor {
+        return object : PhpElementVisitor() {
+            override fun visitPhpElement(element: PhpPsiElement?) {
+                if (element is Declare && element.text.contains(strictTypeEnable)) {
```

Comment:
Not the best way. If there is `strict_type = 1`, then this method will not work. Look at the function `PhpCodeInsightUtil.getStrictTypeDirectiveValue()`

Reply:
Fixed